piktorlog/src/store/index.js

```diff
@@ -1,15 +1,20 @@
-import { applyMiddleware, createStore } from 'redux';
+import { applyMiddleware, createStore, compose } from 'redux';
 import logger from 'redux-logger';
 import thunk from 'redux-thunk';
 // import { middlewareOne, middlewareTwo } from './middleware';
 import rootReducer from './reducers';
 
+// Allows the redux devtools in the browser to read the state. Don't worry about it.
```

Comment:
🤣love the comment

Reply:
Just a quick question. What does `compose` do? Is that to what the comment on 7 is referring

Reply:
Quick question: What does `compose` on line 1 do? I might just need to review redux.